lib/valid_tags.rb

```diff
@@ -0,0 +1,79 @@
+class ValidTags
```

Comment:
Seeing how everyone _loves_ YAML, would it be useful to have this as a configuration file? It probably won't add much, but would allow people to configure the app without changing code.

Reply:
Yeah, if having this in Yaml becomes a requirement we can add that later.